build/Common.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <LangVersion>9.0</LangVersion>
+    <LangVersion>10.0</LangVersion>
```

Comment:
@cijothomas - Since this is a small change I included it here. Let me know if I should create a separate pr for this.

Reply:
lets keep it separate as this file affects entire projects, not just persistentstorage. you may do it freely in persistent storage project alone in this PR.